src/main/groovy/io/seqera/wave/service/builder/BuildStrategy.groovy

```diff
@@ -55,35 +55,42 @@ abstract class BuildStrategy {
     protected List<String> dockerLaunchCmd(BuildRequest req) {
         final result = new ArrayList(10)
         result
-                << "--dockerfile"
-                << "$req.workDir/Containerfile".toString()
-                << "--context"
-                << "$req.workDir/context".toString()
-                << "--destination"
-                << req.targetImage
-                << "--cache=true"
-                << "--custom-platform"
-                << req.platform.toString()
+                << "build"
+                << "--frontend"
+                << "dockerfile.v0"
+                << "--local"
+                << "dockerfile=$req.workDir".toString()
+                << "--opt"
+                << "filename=Containerfile"
+                << "--local"
+                << "context=$req.workDir/context".toString()
+                << "--output"
+                << "type=image,name=$req.targetImage,push=true".toString()
+                << "--opt"
+                << "platform=$req.platform".toString()
 
         if( req.cacheRepository ) {
-            result << "--cache-repo" << req.cacheRepository
-        }
-
-        if( !buildConfig.compressCaching ){
-            result << "--compressed-caching=false"
+            result << "--export-cache"
+            def cache = "type=registry,image-manifest=true,ref=$req.cacheRepository:$req.containerId,mode=max,ignore-error=true".toString()
+            if( buildConfig.compressCaching ){
+                cache += ',force-compression=true'
```

Comment:
Wonder if this should be `compression=gzip`

Reply:
that is by default
here is doc
```
oci-mediatypes=<true|false>: whether to use OCI mediatypes in exported manifests (default: true, since BuildKit v0.8)
compression=<uncompressed|gzip|estargz|zstd>: choose compression type for layers newly created and cached, gzip is default value. estargz and zstd should be used with oci-mediatypes=true
compression-level=<value>: choose compression level for gzip, estargz (0-9) and zstd (0-22)
force-compression=true: forcibly apply compression option to all layers
```

Reply:
Not sure instead what `force-compression=true` does? maybe that option can be removed, that was related to a kaniko option I think more related to the build optimisation

Reply:
In any case let's add an option in BuildConfig to control both `oci-mediatypes` and `compression` settings